crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -312,7 +312,10 @@ fn visibility_propagate_system(
             Visibility::Hidden => false,
             Visibility::Inherited => match parent {
                 None => true,
-                Some(parent) => visibility_query.get(parent.get()).unwrap().1.get(),
+                Some(parent) => visibility_query
+                    .get(parent.get())
+                    .map(|x| x.1.get())
+                    .unwrap_or(true),
             },
```

Comment:
Should be able to simplify this arm to

```rust
Visibility::Inherited => parent
    .and_then(|p| visibility_query.get(p.get()).ok())
    .map_or(true, |x| x.1.get())
```

Reply:
Maybe with a comment too to speed comprehension :p